Find-Package.NuGet.ArgumentCompleters.ps1

```diff
@@ -0,0 +1,84 @@
+<#PSScriptInfo
+.VERSION 0.0.1
+.AUTHOR brywang-msft
```

Comment:
>-msft [](start = 15, length = 5)

Consider Author name as 'Microsoft Corporation' or 'PowerShell Team'

Reply:
Changing to PowerShell Team

---
In reply to: [154492145](https://github.com/OneGet/NuGetProvider/pull/25#discussion_r154492145) [](ancestors = 154492145)